### PR TITLE
Configure ci operator for odf-console

### PR DESCRIPTION
**Pull request by vbnrh**
This commit adds commands to build the odf-operator with odf-console.

Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 1]
```diff
@@ -1,18 +1,81 @@
...
+    dependencies:
+      OO_INDEX: ci-index-odf-operator-master
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-storage
+      OO_PACKAGE: odf-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: tests
+      commands: BRIDGE_KUBEADMIN_PASSWORD="$(cat /tmp/artifacts/installer/auth/kubeadmin-password)";
```
Comment (rexagod) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:68]: We'll need to tweak the config here a bit (similar to https://github.com/openshift/release/blob/be73d9636886da0f5955a7712101d7ab8817fda3/ci-operator/config/openshift/console/openshift-console-master.yaml#L104) to allow the prow script injection for https://github.com/red-hat-storage/odf-console/pull/38 to work.

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 2]
```diff
@@ -13,6 +59,36 @@ tests:
...
+      OO_INDEX: ci-index-odf-operator-master
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-storage
+      OO_PACKAGE: odf-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: tests
+      cli: latest
+      commands: mkdir cypress/tests  ; echo 'describe("Test Intallation", () => {
```
Comment (bipuladh) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:77]: Remove this

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 3]
```diff
@@ -1,6 +1,52 @@
...
+    dockerfile_path: ocsbundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/ocs-dev/odf-operator:latest
+    with: pipeline:odf-operator
+  - pullspec: quay.io/ocs-dev/odf-console:latest
+    with: pipeline:odf-console
+releases:
+  initial:
+    integration:
+      name: "4.10"
```
Comment (bipuladh) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:43]: Use 4.9 until 4.10 issue resolves.

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 4]
```diff
@@ -13,6 +59,36 @@ tests:
...
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-storage
+      OO_PACKAGE: odf-operator
+      OO_TARGET_NAMESPACES: '!install'
+      ZONES_COUNT: "1"
+    test:
+    - as: tests
+      cli: latest
+      commands: ./test-prow-e2e.sh ${ODF_CONSOLE_FULL_IMG_NAME} ; oc get storagecluster
+        ocs-storagecluster -n openshift-storage -o yaml ;
```
Comment (bipuladh) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:80]: why are we pulling the yaml of the storage cluster?

**Pull request merged**